src/static/index.html

```diff
@@ -179,7 +179,7 @@
           .getUnprocessedInputRows();
       });
       const inputHeaders = unprocessedInputRows.shift();
-      const maxBatchSize = 30;
+      const maxBatchSize = 10;
```

Comment:
why? I think it's because of all those unnecessary retries up there. I would be in favor of keeping it 30 to keep FeedGen as fast as it can be.

Reply:
reverting to 30, as discussed.